templates/zerver/for/companies.md

```diff
@@ -0,0 +1,68 @@
+Zulip's threading allows asynchronous discussion to happen over chat, which
+completely changes what chat can be used for in a company. Zulip's threading
+allows managers to weigh in on discussions even when they can only check in
+once a day, it allows workers to participate in discussions even when they
+live in different time zones, and it allows neighboring teams to keep tabs
+on conversations that might affect them.
+
+Communication is at the core of the operation of every large business. Many
+senior personnel spend most of their days in meetings, and many decisions
+have to bubble up a management chain because employees on the ground don't
+have enough context. Employers also lose out on remote talent because
+communication with remote team members is too burdensome with their existing
+tools.
+
+No chat product comes even close to Zulip in terms of enabling serious
+discussion, engaging remote participants, spreading knowledge to neighboring
+teams, and making efficient use of managers' time.
+
+If you haven't read [why Zulip](/why-zulip), read that first. We've also
+collected a list of features we think will be of particular interest for
+companies using Zulip.
+
+**On-premise and in the cloud**
+
+High quality export and import tools make it easy to start with Zulip in the
+cloud, knowing that you can move to a on-premise deployment (or back) if
+your budget or security needs change.
+
+**No vendor lock-in**
+
+Zulip's license (Apache 2) and [published code base](github.com/zulip/zulip)
+means you will always be able to run Zulip on your own servers, for free,
+forever.
+
+**Highly customizable**
+
+Zulip has a well-designed API, over 100,000 words of developer
+documentation, over 90% test coverage, and full static typing of our python
+codebase.
+
+Previous customers have added LaTeX support for discussing academic papers,
+authentication protocols for integrating with a custom SSO, and an
+artificial message delay feature for emergency training. Think of a product
+you use every day at work. What would you change about it if your dev ops
+team could have a go at it for a week?
+
+**Scale to thousands of users**
+
+Zulip teams are still snappy at thousands of users.
+
+**Hundreds of integrations**
+
+Get events from GitHub, Stripe, Travis CI, JIRA, and
+[hundreds of other tools](/integrations) right in Zulip. Use topics to give
+each issue its own place for discussion. Link to tickets in external sites
+with [custom linkification filters](/help/add-a-custom-linkification-filter)
+like `Z1234` for Zendesk ticket #1234.
+
+**Rich message formatting**
+
+Express your ideas clearly with bulleted lists, code blocks, pasted
+screenshots, uploaded files, and more. Zulip has
+[full Markdown support](/help/format-your-message-using-markdown).
```

Comment:
`s/full Markdown support/chat optimized Markdown/`; we turn off some things like headings, and add a bunch more like ` ``` quote`.